cmake/ROCMExportTargetsHeaderOnly.cmake

```diff
@@ -95,11 +95,12 @@ function(rocm_export_targets_header_only)
 
     if(PARSE_TARGETS)
         rocm_write_package_template_function(${CONFIG_TEMPLATE} include "\${${PACKAGE_NAME}_TARGET_FILE}")
-# Disabled for PyTorch
-#        foreach(NAME ${PACKAGE_NAME} ${PACKAGE_NAME_UPPER} ${PACKAGE_NAME_LOWER})
-#            rocm_write_package_template_function(${CONFIG_TEMPLATE} set ${NAME}_LIBRARIES ${PARSE_TARGETS})
-#            rocm_write_package_template_function(${CONFIG_TEMPLATE} set ${NAME}_LIBRARY ${PARSE_TARGETS})
-#        endforeach()
+        if(NOT (DEFINED PYTORCH_FOUND_HIP AND PYTORCH_FOUND_HIP) )
```

Comment:
rocm_write_package_template_function(${CONFIG_TEMPLATE} if(NOT (DEFINED PYTORCH_FOUND_HIP AND PYTORCH_FOUND_HIP) ) ) something like this as the switch needs to be in the file written

Reply:
The problem is in the file that is search by the PyTorch cmake.

Reply:
Offending file is installed in /opt/rocm/rocthrust/lib/cmake/rocthrust/rocthrust-targets.cmake

Reply:
I think it's solved.